liquibase-standard/src/main/java/liquibase/parser/AbstractFormattedChangeLogParser.java

```diff
@@ -717,6 +719,19 @@ protected boolean handleAdditionalLines(DatabaseChangeLog changeLog, ResourceAcc
         return false;
     }
 
+    //
+    // If the change set ID is empty after removing colons and blank spaces then it is invalid
+    //
```

Comment:
Can you make this an actual javadoc not a comment?

Reply:
Yes